app/views/products/_form.html.erb

```diff
@@ -11,24 +11,39 @@
   </div>
   <% end %>
   
+   <div class="field">
```

Comment:
You have added all these fields to the form but do they get saved to db when you submit the form?